geowebcache/diskquota/core/src/main/java/org/geowebcache/diskquota/UsageStatsMonitor.java

```diff
@@ -74,10 +74,14 @@ public void startUp() {
         executorService.submit(usageStatsConsumer);
 
         usageStatsProducer = new QueuedUsageStatsProducer(sharedQueue);
-        Iterable<TileLayer> allLayers = tileLayerDispatcher.getLayerList();
-        for (TileLayer layer : allLayers) {
-            layer.addLayerListener(usageStatsProducer);
-        }
+
+        
+        // Will be added to layer when used, see DiskQuotaMonitor#getLayerListener
+        // Iterable<TileLayer> allLayers = tileLayerDispatcher.getLayerList();
+        // for (TileLayer layer : allLayers) {
+        // layer.addLayerListener(usageStatsProducer);
+        // }
```

Comment:
Again, please don't leave commented out code unless it's an example or something like that.